src/gt4py/cartesian/backend/dace_backend.py

```diff
@@ -12,7 +12,7 @@
 import os
 import pathlib
 import re
-from typing import TYPE_CHECKING, Any, ClassVar, Dict, List, Optional, Tuple, Type
+from typing import TYPE_CHECKING, Any, ClassVar, Type
```

Comment:
Same as above (for all files in the PR)
```suggestion
from typing import TYPE_CHECKING, Any, ClassVar,
```